frontend/src/components/MultiSearch.tsx

```diff
@@ -87,17 +87,21 @@ const MultiSearchBar: React.FC<MultiSearchBarProps> = ({
     const handleKeyDown = (event: KeyboardEvent) => {
       if (event.key === 'Escape') {
         setShowSuggestions(false)
-      } else if (event.key === 'Enter' && highlightedIndex !== null) {
-        const { index, subIndex } = highlightedIndex
-        const suggestion = suggestions[index].hits[subIndex]
-        handleSuggestionClick(suggestion, suggestions[index].indexName)
+      }
+      else if (event.key === 'Enter' && highlightedIndex !== null) {
+        const { index, subIndex } = highlightedIndex as { index: number, subIndex: number };
+        const suggestion = suggestions[index]?.hits?.[subIndex];
+      
+        if (suggestion) {
+          handleSuggestionClick(suggestion, suggestions[index].indexName);
+        }
       } else if (event.key === 'ArrowDown') {
         event.preventDefault()
         if (highlightedIndex === null) {
           setHighlightedIndex({ index: 0, subIndex: 0 })
         } else {
-          const { index, subIndex } = highlightedIndex
-          if (subIndex < suggestions[index].hits.length - 1) {
+          const { index, subIndex } = highlightedIndex  ;      
+           if (subIndex < suggestions[index].hits.length - 1) {
```

Comment:
Can we clean this up? The spaces before `;` and the extra space before `if` statement? 
You should run an auto-formatter that would fix these for you.

Reply:
Also, is it guaranteed that `suggestions[index].hits` is available? Do we need to add `?` maybe?